--- gnomeradio-1.8/src/gui.c.thread	2008-09-13 17:49:07.000000000 +0200
+++ gnomeradio-1.8/src/gui.c	2009-03-15 11:15:32.000000000 +0100
@@ -1096,6 +1096,7 @@
 #if GNOME_14
     if (!g_thread_supported ()) g_thread_init(NULL);
 	ctx = g_option_context_new("- Gnomeradio");
+        if (!g_thread_supported ()) g_thread_init(NULL);
 	g_option_context_add_main_entries(ctx, entries, GETTEXT_PACKAGE);  
 	g_option_context_add_group(ctx, gst_init_get_option_group());
 	g_option_context_set_ignore_unknown_options(ctx, TRUE);	
